### DISCUSSION TITLE
Ubuntu 23.10 works without upgrading zfsbootmenu

### DISCUSSION THREAD

**Discussion by vitali-fridman** [zbm-dev/zfsbootmenu#507] [General]
Ubuntu 23.10 has zfs version 2.2.0.rc3. I thought it requires updating zfsbootmenu to the latest. However while upgrading Ubuntu on several systems I forgot to upgrade zfsbootmenu on one of them. And despite of it everything works just fine. It also works fine with the latest zfsbootmenu as expected.

**Comment by zdykstra** [comment]
Previous source releases of ZFSBootMenu work fine with ZFS 2.2.0. ZFSBootMenu 2.2.1 was tagged so that binaries could be built with ZFS 2.2.0. This is a hard requirement for upgraded pools.
